test/k8sT/manifests/demo.yaml

```diff
@@ -14,7 +14,12 @@ metadata:
   name: app1-service
 spec:
   ports:
-  - port: 80
+  - name: http
```

Comment:
Maybe add a comment saying that changing the order of ports will make the `test/k8sT/Services.go` to fail?

Reply:
Or maybe even introduce a "getServicePortByName" helper, since we name them consistently. I might do this in a follow up PR.